src/test/java/api/loans/DueDateScheduledNoticesTests.java

```diff
@@ -544,4 +549,104 @@ public void noticesShouldBeRescheduledAfterManualDueDateChange() {
+      noticePoliciesFixture.activeNotice().getId(),
+      overdueFinePoliciesFixture.facultyStandardDoNotCountClosed().getId(),
+      lostItemFeePoliciesFixture.facultyStandard().getId());
+
+    UUID uponAtTemplateId = UUID.randomUUID();
+
+    UUID afterTemplateId = UUID.randomUUID();
+    Period afterPeriod = Period.days(3);
+    Period afterRecurringPeriod = Period.hours(4);
+
+    JsonObject uponAtDueDateNoticeConfiguration = new NoticeConfigurationBuilder()
+      .withTemplateId(uponAtTemplateId)
+      .withDueDateEvent()
+      .withUponAtTiming()
+      .sendInRealTime(false)
+      .create();
+    JsonObject afterDueDateNoticeConfiguration = new NoticeConfigurationBuilder()
+      .withTemplateId(afterTemplateId)
+      .withDueDateEvent()
+      .withAfterTiming(afterPeriod)
+      .recurring(afterRecurringPeriod)
+      .sendInRealTime(true)
+      .create();
+
+    NoticePolicyBuilder noticePolicy = new NoticePolicyBuilder()
+      .withName("Policy with due date notices")
+      .withLoanNotices(Arrays.asList(
+        uponAtDueDateNoticeConfiguration,
+        afterDueDateNoticeConfiguration));
+    use(noticePolicy);
+
+    final IndividualResource james = usersFixture.james();
+    final UUID checkInServicePointId = servicePointsFixture.cd1().getId();
+    final IndividualResource homeLocation = locationsFixture.basedUponExampleLocation(
+      item -> item.withPrimaryServicePoint(checkInServicePointId));
+    final IndividualResource nod = itemsFixture.basedUponNod(item ->
+      item.withPermanentLocation(homeLocation.getId()));
+    DateTime loanDate = new DateTime(2020, 1, 1, 12, 0, 0, UTC);
+
+    final IndividualResource loan = checkOutFixture.checkOutByBarcode(
+      new CheckOutByBarcodeRequestBuilder()
+        .forItem(nod)
+        .to(james)
+        .on(loanDate)
+        .at(UUID.randomUUID()));
+
+    DateTime dueDate = getDateTimeProperty(loan.getJson(), "dueDate");
+
+    Awaitility.await()
+      .atMost(1, TimeUnit.SECONDS)
+      .until(scheduledNoticesClient::getAll, hasSize(2));
+
+    List<JsonObject> scheduledNotices = scheduledNoticesClient.getAll();
+    assertThat(scheduledNotices,
+      hasItems(
+        hasScheduledLoanNotice(
+          loan.getId(), dueDate,
+          UPON_AT_TIMING, uponAtTemplateId,
+          null, false),
+        hasScheduledLoanNotice(
+          loan.getId(), dueDate.plus(afterPeriod.timePeriod()),
+          AFTER_TIMING, afterTemplateId,
+          afterRecurringPeriod, true)
+      ));
+
+    JsonObject servicePointOwner = new JsonObject();
+    servicePointOwner.put("value", homeLocation.getJson().getString("primaryServicePoint"));
+    servicePointOwner.put("label", "label");
+    UUID ownerId = UUID.randomUUID();
+    feeFineOwnersClient.create(new FeeFineOwnerBuilder()
+      .withId(ownerId)
+      .withOwner("fee-fine-owner")
+      .withServicePointOwner(Collections.singletonList(servicePointOwner))
+    );
```

Comment:
There is `api.support.fixtures.FeeFineOwnerFixture` could it be moved to this class?

We can introduce new method `ownerForServicePoint(String spId)`

Reply:
Done.